docs/resources/grants.md

```diff
@@ -3,20 +3,40 @@ subcategory: "Unity Catalog"
 ---
 # databricks_grants Resource
 
--> **Public Preview** This feature is in [Public Preview](https://docs.databricks.com/release-notes/release-types.html). Contact your Databricks representative to request access. 
+-> **Public Preview** This feature is in [Public Preview](https://docs.databricks.com/release-notes/release-types.html). Contact your Databricks representative to request access.
 
 In Unity Catalog all users initially have no access to data. Only Metastore Admins can create objects and can grant/revoke access on individual objects to users and groups. Every securable object in Unity Catalog has an owner. The owner can be any account-level user or group, called principals in general. The principal that creates an object becomes its owner. Owners receive all privileges on the securable object (e.g., `SELECT` and `MODIFY` on a table), as well as the permission to grant privileges to other principals.
-
+<!--This matches https://github.com/databricks/docs/blob/master/source/shared/unity-catalog/privileges.md-->
 Unity Catalog supports the following privileges on securable objects:
-* `SELECT` - Allows the grantee to read data from the securable (applicable to tables and views).
-* `MODIFY` - Allows the grantee to add, update and delete data to or from the securable. (applicable to tables)
-* `CREATE` - Allows the grantee to create child objects within this securable.
-* `USAGE` - This privilege does not grant access to the securable itself, but allows the grantee to traverse the securable in order to access its child objects. For example, to select data from a table, users need to have the `SELECT` privilege on that table and `USAGE` privileges on its parent schema and parent catalog. Thus, you can use this privilege to restrict access to sections of your data namespace to specific groups.
+
+- `USAGE`: Applicable object types: CATALOG, SCHEMA. This privilege does not grant access to the securable itself, but is needed for a user to interact with any object within the securable. For example, to select data from a table, users need to have the SELECT privilege on that table and USAGE privileges on its parent schema and parent catalog.
+  
```

Comment:
can we have no blank lines between list items? how does this render on https://registry.terraform.io/tools/doc-preview?

Reply:
markdown will still render correctly, but will remove them